src/components/DefineCompany/YourCompany.vue

```diff
@@ -11,14 +11,8 @@
           <v-layout column>
             <label><strong>Company Name</strong></label>
             <v-flex md1>
-              <v-chip
-                v-if="companyNameChanges"
-                class="ma-2 pa-2"
-                color="primary"
-                pill
-                x-small
-              >
-                Corrected
+              <v-chip x-small label color="primary" text-color="white" id='corrected-lbl'>
```

Comment:
Please use " for strings in the template.

Reply:
Also, I think text-color doesn't need to be set here (white by default).

Reply:
_Update:_ When color is "primary" then the text is automatically white, but if you set color to, say, "blue", then you have to set the text color also.
So here, the text color is redundant.

Reply:
_Update2:_ The design specifies a colour of #1669BB aka rgb(22, 105, 187).

Please update all references.

Reply:
Sure. Will do that